src/tqec/computation/block_graph_test.py

```diff
@@ -229,8 +230,8 @@ def test_graph_rotation() -> None:
 
 @pytest.mark.parametrize("obs_basis", [Basis.Z, Basis.X, None])
 def test_cnot_graph_rotation(obs_basis: Basis | None) -> None:
-    from tqec.interop.collada.read_write_test import rotated_cnot
     from tqec.gallery.cnot import cnot
+    from tqec.interop.collada.read_write_test import rotated_cnot
```

Comment:
Is there a particular reason to use [deferred imports](https://peps.python.org/pep-0690/) for this unit test?

Reply:
I don't think. These can likely go with the other imports, except if we have a circular dependency on test files (which would be surprising)

Reply:
Well, that's the thing. I checked these locally, and there were no circular import errors after I moved the imports. I could not tell if there was another reason for it....

Reply:
Let's do the change, and see if anything breaks. I expect this is just something that slipped through PR and I do not see any way this could impact the code outside of circular imports (that should raise directly)

Reply:
Yup, made the change in https://github.com/tqec/tqec/pull/599/commits/728bcee26e8bed9060a6ae58c64b35c851ac7a23. There were no failures. :)